.gitignore

```diff
@@ -26,6 +26,7 @@ share/
 
 # Editors' temporary files
 .*.swp
+.idea
```

Comment:
Nitpick: you should add this rule to your `$ git config core.excludesfile`. 😉

Reply:
Thanks for the tip!